internal/api/api.go

```diff
@@ -144,8 +145,35 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, database db.S
 		ErrorHandler: func(_ *fiber.Ctx, err error) error {
 			return fiber.NewError(fiber.StatusUnauthorized, "Invalid JWT. "+err.Error())
 		},
+		SuccessHandler: func(_ *fiber.Ctx) error {
```

Comment:
just want to check this will still work with regular auth JWT - we can test in dev

Reply:
Let's test it.